mani/utils.py

```diff
@@ -1,2 +1,82 @@
-"""Add your final methods here."""
+import random
+import webbrowser
+import time
+
+# Defining the items
+
+fruits = ["apple", "banana", "mango"]
+vegs = ['carrot', 'beans', 'potato']
+drinks = ['milk', 'water', 'juice']
+
+items = fruits+vegs+drinks
+
+fruit_url="https://en.wikipedia.org/wiki/Fruit"
+
+drink_url="https://en.wikipedia.org/wiki/Drink"
+
+veg_url="https://en.wikipedia.org/wiki/Vegetable"
+
+def what_is_the_item(category):
+      if category in fruits:
+            print("\n", category + " is a fruit")
+            time.sleep(3)
+            webbrowser.open(fruit_url)
+      elif category in drinks:
+            print("\n", category + " is a drink")
+            time.sleep(3)
+            webbrowser.open(drink_url)
+      else:
```

Comment:
change else >>> elif 


and else at the bottom. (this will print("Item not Valid")  if user passing items not in the list.

Reply:
yes we can do that, sorry missed it